kvbc/src/kvbc_adapter/kv_blockchain_adapter.cpp

```diff
@@ -0,0 +1,85 @@
+// Concord
+//
+// Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#include "kvbc_adapter/kv_blockchain_adapter.hpp"
+#include "assertUtils.hpp"
+
+namespace concord::kvbc::adapter {
+
+KeyValueBlockchain::KeyValueBlockchain(
+    const std::shared_ptr<concord::storage::rocksdb::NativeClient> &native_client,
+    bool link_st_chain,
+    const std::optional<std::map<std::string, categorization::CATEGORY_TYPE>> &category_types,
+    const std::optional<aux::AdapterAuxTypes> &aux_types)
+    : kvbc_(std::make_unique<concord::kvbc::categorization::KeyValueBlockchain>(
```

Comment:
same comment as IAppState

Reply:
Thanks a lot @cloudnoize , you save me. This is really great comment, helped me a lot.